### PR TITLE
Transfer to Es modules

### PR DESCRIPTION
**Pull request by jennyliang220**
resolves https://github.com/w3c/echidna/issues/1099
close #1130

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -214,7 +221,7 @@ const processRequest = (req, res, isTar) => {
         // eslint-disable-next-line no-console
         if (dryRun) console.log('Dry-run: omitting e-mail notification');
         else {
-          mailer.sendMessage(
+          await sendMessage(
```
Comment (deniak) [inline app.js:224]: is there any reason to use `await` here?

File: config-dev.js [hunk 2]
```diff
@@ -1,45 +1,49 @@
...
-  /https?:\/\/w3c\.github\.io/i,
-];
-global.LDAP_URL = 'ldap://localhost:1389';
-global.LDAP_SEARCH_BASE = 'ou=user,dc=example,dc=org';
-// LDAP_BIND_DN must contain the placeholder {{username}}
-global.LDAP_BIND_DN = 'uid={{username}},ou=user,dc=example,dc=org';
-global.GH_TOKEN = '';
-global.GH_DIRECTOR_TEAM_ID = '2797096';
-global.GH_COMM_TEAM_ID = '2794457';
+function setGlobalInfo() {
```
Comment (deniak) [inline config-dev.js:3]: This is the dev config. In production, we are using a different config file based from `config.js.example`.

**Review by deniak — CHANGES_REQUESTED** [review]

File: lib/history.js [hunk 3]
```diff
@@ -4,14 +4,17 @@
...
+import { importJSON } from './util.js';
 
-const config = process.env.CONFIG || 'config.js';
+const meta = importJSON('../package.json', import.meta.url);
+const { Stack } = pkg;
+
+// const config = process.env.CONFIG || 'config.js';
 // eslint-disable-next-line import/no-dynamic-require
-require(`../${config}`);
+// import(`../${config}`);
```
Comment (deniak) [inline lib/history.js:17]: delete comments

File: lib/orchestrator.js [hunk 4]
```diff
@@ -1,26 +1,24 @@
...
+import IPChecker from './ip-checker.js';
+
+const { List, Map } = pkg;
 
 // Configuration file
-const config = process.env.CONFIG || 'config.js';
+// const config = process.env.CONFIG || 'config.js';
 // eslint-disable-next-line import/no-dynamic-require
-require(`../${config}`);
+// import(`../${config}`);
```
Comment (deniak) [inline lib/orchestrator.js:21]: delete comments

**Review by deniak — CHANGES_REQUESTED** [review]

**Comment by jennyliang220** [comment]
## [Import a module for its side effects only](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/import#import_a_module_for_its_side_effects_only)

```
(async () => {
  if (somethingIsTrue) {
    // import module for side effects
    await import("/modules/my-module.js");
  }
})();

```

This could work for the global variables.

**Review by deniak — APPROVED** [review]

**Pull request merged**